public/index.html

```diff
@@ -5,7 +5,21 @@
       <meta charset="utf-8">
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
       <link rel="shortcut icon" href="<%=htmlWebpackPlugin.options.FAVICON_URL%>" type="image/x-icon" />
-  </head>
+    <!-- Hotjar Tracking Code for https://www.edx.org/ -->
+    <script>
+      const isMobile = window.navigator.userAgent.includes("org.edx.mobile");
```

Comment:
I don't think we should care about mobile for this.

Reply:
Once we enable mobile this code snippet may cause issues so being proactive.